### PR TITLE
[terraform] remote terraform state

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

**Comment by danking** [comment]
As part of your review, can you try to load this remote state and `terraform plan` and verify nothing has changed?

File: infra/gcp-broad/README.md [hunk 1]
```diff
@@ -3,3 +3,86 @@ If you're a third-party trying to deploy Hail, look at `../gcp`.
...
+cat >infra/gcp-broad/$GITHUB_ORGANIZATION/backend.hcl <<EOF
+bucket = $TERRAFORM_STATE_BUCKET
+kms_encryption_key = $TERRAFORM_KEY_NAME
+EOF
+```
+
+Initialize Terraform:
+
+```
+terraform init -backend-config=hail-is/backend.hcl -var-file=hail-is/global.tfvars
```
Comment (daniel-goldstein) [inline infra/gcp-broad/README.md:81]: Did you mean to check these in but didn't? (I'm assuming they're encrypted) I checked out this branch and don't see a `hail-is` dir

**Review by daniel-goldstein — APPROVED** [review]

**Review by daniel-goldstein — CHANGES_REQUESTED** [review]
Don't have the backend.hcl and global.tfvars

**Review by daniel-goldstein — DISMISSED** [review]
Don't have the backend.hcl and global.tfvars

File: infra/gcp-broad/README.md (hunk 1 above)
Comment (danking) [reply infra/gcp-broad/README.md:81]: Hmm. I intentionally didn't. They're not encrypted. There's no secret data in them. There is the *name of* a secret. I suppose we need to distribute these files to hail developers somehow. Seemed a bit odd to include them in the repo, but I guess that's the simplest place for them.

**Comment by danking** [comment]
Whoops I didn't check it in last time. it is there now.

**Comment by danking** [comment]
bump

**Comment by daniel-goldstein** [comment]
I'm getting ~13 changes when I `terraform plan` now. Most are related to terraform wanting to replace the network because `auto_create_subnetworks` is false and it wants it to be true. It's also complaining about no `name` label on `hail_test_requester_pays_bucket` and it is not currently aware of `zulip_config` and wants to recreate it. I'm guessing the zulip config is also a file on your machine that needs to go through SOPS and be committed?

**Review by daniel-goldstein — DISMISSED** [review]
See comment

**Comment by danking** [comment]
Mmm, my bad. I added the tag to the bucket without changing terraform. I'll add to this PR.

**Comment by danking** [comment]
I'm not sure how I flubbed the network. I had to do that manually to avoid recreating everything. I should've verified the terraform was still accurate afterwards.

**Comment by danking** [comment]
Ah, you must have a file at `~/.hail/.zuliprc`. This should really live inside hail-is, right? I've made it a SOPS encoded file, placed it in hail-is, and imported the secret.

This is what it was before:
```
resource "kubernetes_secret" "zulip_config" {
  count = fileexists("~/.hail/.zuliprc") ? 1 : 0
  metadata {
    name = "zulip-config"
  }

  data = {
    ".zuliprc" = fileexists("~/.hail/.zuliprc") ? file("~/.hail/.zuliprc") : ""
  }
}
```

**Review by daniel-goldstein — APPROVED** [review]
No changes in the plan!

**Pull request merged**